.github/workflows/test_and_deploy.yml

```diff
@@ -14,62 +14,78 @@ jobs:
         with:
           node-version: lts/iron
 
+        # We cannot use build in LFS due to missing support of custom URLs: https://github.com/actions/checkout/issues/365
```

Comment:
This sentence feels a bit weird, I don't understand it. What do you mean with "cannot use build in LFS"?